CHANGELOG.md

```diff
@@ -1,3 +1,118 @@
+# CHANGELOG 1.9.0
+
+# v1.9.0 (2022-07-28)
+
+## OS Changes
+
+### General
+
+* SELinux policy now suppresses audit for tmpfs relabels ([#2222])
+* Restrict permissions for /boot and System.map ([#2223])
+* Remove unused crates growpart and servicedog ([#2238])
+* Rust dependency updates ([#2303])
+* Go dependency updates ([#2299])
+* Migrate to using `aws-sdk-rust` for first-party rust packages ([#2300])
+* New mount in host containers for system logs ([#2295])
+* Add support for AWS China regions ( [#2224], [#2242], [#2247], [#2285])
+* Apply strict mount options and enforce execution rules ([#2239])
+* Clean up uncommon way of disabling kernel config options ([#2290])
```

Comment:
nit. It might be easier to follow if we say "Switch to a more commonly used syntax for disabling kernel config settings"